src/GraphQL.SystemTextJson/StringExtensions.cs

```diff
@@ -36,5 +40,81 @@ public static Inputs ToInputs(this string json)
         /// <returns>Dictionary.</returns>
         public static Dictionary<string, object> ToDictionary(this string json)
             => JsonSerializer.Deserialize<Dictionary<string, object>>(json, _jsonOptions);
+
+        /// <summary>
+        /// Deserializes a JSON-formatted string of data into the specified type.
+        /// Any <see cref="Dictionary{TKey, TValue}">Dictionary&lt;string, object&gt;</see> objects will be deserialized
+        /// into the proper format for passing to <see cref="InputsExtensions.ToInputs(Dictionary{string, object})"/>.
+        /// </summary>
+        public static T FromJson<T>(this string json)
+            => JsonSerializer.Deserialize<T>(json, _jsonOptions);
+
+        /// <summary>
+        /// Deserializes a JSON-formatted stream of data into the specified type.
+        /// Any <see cref="Dictionary{TKey, TValue}">Dictionary&lt;string, object&gt;</see> objects will be deserialized
+        /// into the proper format for passing to <see cref="InputsExtensions.ToInputs(Dictionary{string, object})"/>.
+        /// </summary>
+        public static ValueTask<T> FromJsonAsync<T>(this System.IO.Stream stream, CancellationToken cancellationToken = default)
+            => JsonSerializer.DeserializeAsync<T>(stream, _jsonOptions, cancellationToken);
+
+        /// <summary>
+        /// Converts a JSON element into an <see cref="Inputs"/> object.
+        /// </summary>
+        public static Inputs ToInputs(this JsonElement obj)
```

Comment:
The corresponding method from Newtonsoft project returns `Inputs.Empty` if `obj` is `null`.

Reply:
Updated that and also added tests.

Reply:
If `obj` is `null` then this method throws NRE.

Reply:
n/a - `JsonElement` is a struct here.

Reply:
🤦‍♂️